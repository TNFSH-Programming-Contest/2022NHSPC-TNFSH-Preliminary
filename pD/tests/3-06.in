7 6
2 1 0 3 0 1 2
3 0 3 0 2 1
...T..
T.....
....TT
.T.T..
..T...
......
.T...T
