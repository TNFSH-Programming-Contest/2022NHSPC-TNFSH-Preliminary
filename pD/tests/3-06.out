A.AT..
T....A
....TT
ATATA.
..T...
..A...
AT..AT
